settings.gradle

```diff
@@ -1 +1,3 @@
 rootProject.name = 'toloka-java-sdk'
+include 'examples'
```

Comment:
please remove, it is not part of main gradle project

Reply:
Done